src/components/QuickOpenModal.js

```diff
@@ -381,9 +371,14 @@ export class QuickOpenModal extends Component<Props, State> {
     const newResults = results && results.slice(0, 100);
     const items = this.highlightMatching(query, newResults || []);
     const expanded = !!items && items.length > 0;
-    const summaryMsg = this.isGotoQuery()
-      ? L10N.getStr("shortcuts.gotoLine")
-      : "";
+    const isLoading =
+      (this.isFunctionQuery() || this.isVariableQuery()) && symbolsLoading;
+    let summaryMsg = "";
+    if (this.isGotoQuery()) {
+      summaryMsg = L10N.getStr("shortcuts.gotoLine");
+    } else if (isLoading) {
+      summaryMsg = L10N.getStr("loadingText");
+    }
```

Comment:
we might want to move these fields to functions

`isLoading() {}`
`getSummaryMessage() {}`